src/Acme/Bundle/RangeBundle/Factory/RangeFactory.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Acme\Bundle\RangeBundle\Factory;
+
+use Acme\Bundle\RangeBundle\Model\Range;
+
+/**
+ * Creates a range object.
+ *
+ * @author    Damien Carcel (damien.carcel@akeneo.com)
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+class RangeFactory
```

Comment:
No, I create a Range object, like for price and metrics. Then I use the RangeProductValueFactory that call this factory.